mu-plugins/blocks/language-suggest/postcss/style.pcss

```
@@ -0,0 +1,18 @@
+/**
+ * The following styles get applied both on the front of your site
+ * and in the editor.
+ *
+ * Replace them with your own styles or remove the file completely.
+ */
+
+.wp-block-wporg-language-suggest {
+	background-color: #eff2ff;
+	color: #1e1e1e;
+	font-size: 12px;
+	text-align: center;
+}
+
+/* The api returns a div element but that isn't guaranteed. */
+.wp-block-wporg-language-suggest > * {
+	padding: 2px 4px;
```

There's a little more padding in the mockup, and a little less line-height

```suggestion
	line-height: 1;
	text-align: center;
}

/* The api returns a div element but that isn't guaranteed. */
.wp-block-wporg-language-suggest > * {
	padding: 10px 4px;
```